axelrod/tests/strategies/test_grudger.py

```diff
@@ -236,22 +259,105 @@ class TestGeneralSoftGrudger(TestPlayer):
     }
 
     def test_strategy(self):
-        """Test strategy with multiple initial parameters"""
-
-        # Starts by cooperating.
-        self.first_play_test(C)
-
-        # Testing default parameters of n=1, d=4, c=2 (same as Soft Grudger)
-        actions = [(C, D), (D, D), (D, C), (D, C), (D, D), (C, D), (C, C), (C, C)]
-        self.versus_test(axelrod.MockPlayer(actions=[D, D, C, C]), expected_actions=actions)
-
-        # Testing n=2, d=4, c=2
-        actions = [(C, D), (C, D), (D, C), (D, C), (D, D), (D, D), (C, C), (C, C)]
-        self.versus_test(axelrod.MockPlayer(actions=[D, D, C, C]), expected_actions=actions,
-                         init_kwargs={"n": 2})
-
-        # Testing n=1, d=1, c=1
-        actions = [(C, D), (D, D), (C, C), (C, C), (C, D), (D, D), (C, C), (C, C)]
-        self.versus_test(axelrod.MockPlayer(actions=[D, D, C, C]), expected_actions=actions,
-                         init_kwargs={"n": 1, "d": 1, "c": 1})
-
+        # the default follows SoftGrudger
+        super(TestGeneralSoftGrudger, self).test_strategy()
+
```

Comment:
@drvinceknight - in the previous PR you asked me why i changed from the original `test_strategy` to the following three tests.  so I thought it over and here is my reasoning.  

1 - I thought tests physically separated from each other and the default behavior tests were more clear.
2 - for `n != 1` I wanted tests that showed the edges at and below the threshold.  (and what happens with constant defections, which, for reasons unknown to me, I kept getting surprised about)

whether or not these are good reasons, I do think it useful to have this inherit from `TestSoftGrudger` which will test the default behavior and `reset`